cfpb/consumerfinance.gov
# Remove Wagtail override of BooleanColumn

As part modeladmin-to-snippets migration implemented as part of the planned Wagtail 5.0 upgrade in #8038, I brought forward the BooleanColumn class that we wouldn't have access to until Wagtail 5.1.

Then we decided to instead migrate directly to 5.1, but I neglected to remove the override. This PR cleans that up.

## How to test this PR

To test, visit http://localhost:8000/admin/snippets/v1/banner/ and confirm that the boolean column correctly renders as an icon.

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/1d889aae-aa93-4be9-b933-78f7c0888b04)

## Checklist
